arangodb-net-standard/CursorApi/CursorApiClient.cs

```diff
@@ -174,6 +175,7 @@ protected virtual WebHeaderCollection GetHeaderCollection(CursorHeaderProperties
         /// <param name="cursorId">ID of the existing query cursor.</param>
         /// <param name="token">A CancellationToken to observe while waiting for the task to complete or to cancel the task.</param>
         /// <returns></returns>
+        [Obsolete("Use PostCursorAsync")]
```

Comment:
`PostCursorAsync` doesn't provide a way to get more results from a cursor at the moment. Some implementation for the replacement method have been discussed on #326 and so far, creating a  separate method to `PostCursorAsync` is prefered.

Maybe simply use `[Obsolete()]` until a replacement method is available?

Reply:
@DiscoPYF, the solution is to implement another overload of `PostCursorAsync()` as described here https://www.arangodb.com/docs/stable/http/aql-query-cursor-accessing-cursors.html#read-next-batch-from-cursor

Reply:
@DiscoPYF , I've added the `PostCursorAsync` overload and updated the obsolete message on `PutCursorAsync`. Please check. Thanks.

Reply:
> the solution is to implement another overload of PostCursorAsync()

This is one of the solutions, yes. Another approach is to create a method with a different name instead of an overload to make a clearer distinction.

One downside of the overload approach is that it clashes with one of the existing "convenience" overload. The following two methods will conflict if someone calls `PostCursorAsync` with just a query string:

```
Task<PostCursorResponse<T>> PostCursorAsync<T>(
            string query,
            Dictionary<string, object> bindVars = null,
            PostCursorOptions options = null,
            bool? count = null,
            long? batchSize = null,
            bool? cache = null,
            long? memoryLimit = null,
            int? ttl = null,
            string transactionId = null,
            CancellationToken token = default);

Task<PostCursorResponse<T>> PostCursorAsync<T>(string cursorIdentifier, CancellationToken token = default);
```

```
string aql = @"FOR doc IN MyCollection
            FILTER doc.SomeAttribute == true
            RETURN doc";
await adb.Cursor.PostCursorAsync<MyDocument>(aql);
```

```
error CS0121: The call is ambiguous between the following methods or properties: 'CursorApiClient.PostCursorAsync<T>(string, Dictionary<string, object>, PostCursorOptions, bool?, long?, bool?, long?, int?, string, CancellationToken)' and 'CursorApiClient.PostCursorAsync<T>(string, CancellationToken)'
```

Are we ok with this? In real world scenarios, most AQL queries will have bind parameters but for anyone passing only a query string then experience is not great. They have to add a null bindVars `PostCursorAsync<MyDocument>(aql, null);`, which you've had to do in the tests.

An alternative is to add a new method called `PostExistingCursor` or `PostAdvanceCursor`.

I think the overload approach will work fine overall, but I just wanted to make sure we've considered and discussed alternatives.

Reply:
@DiscoPYF , I agree we should use an alternative name; I think PostAdvanceCursor is more appropriate.

Reply:
@DiscoPYF, update to `PostAdvanceCursor` is completed. Please check.